examples/backups/stop_tablets.sh

```diff
@@ -30,7 +30,25 @@ for tablet in 100 200 300; do
       CELL=zone1 TABLET_UID=$uid ../common/scripts/mysqlctl-down.sh
       echo "Removing tablet directory zone1-$uid"
       vtctldclient DeleteTablets --allow-primary zone1-$uid
-      rm -Rf $VTDATAROOT/vt_0000000$uid
+
+      for ((i=0; i<30; i++)); do
+          # Redirect stderr to a temporary file
+          temp_file=$(mktemp)
+          rm -Rf $VTDATAROOT/vt_0000000$uid 2>$temp_file
+
+          if grep -q 'Directory not empty' $temp_file; then
+              echo "Directory not empty, retrying..."
+          elif [ ! -s $temp_file ]; then
```

Comment:
We should quote the `$temp_file` variable everywhere to be safe (can run shellcheck on the file too).

Reply:
Addressed via 8841067156